Specs/DateTime/English/DateTimeModelComplexCalendar.json

```diff
@@ -2194,5 +2200,105 @@
         }
       }
     ]
+  },
+  {
+    "Input":
+      "Cortana, can you schedule a meeting for us between 3:30pm and 6:30pm AEST?",
+    "Context": {
+      "ReferenceDateTime": "2018-05-31T00:00:00"
+    },
+    "NotSupported": "javascript,python",
+    "Results": [
+      {
+        "Text": "between 3:30pm and 6:30pm aest",
+        "Start": 43,
+        "End": 72,
+        "TypeName": "datetimeV2.timerange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(T15:30,T18:30,PT3H)",
+              "type": "timerange",
+              "timezone": "UTC+10:00",
+              "timezoneText": "aest",
```

Comment:
Was the test code modified to check if these are correct? I didn't see it.

Reply:
We also need to be sure these new texts *don't* appear in the standalone timezone entity recognitions.

Reply:
The test code already supports this checking. The new texts don't appear in the standalone timezone resolution, but it seems we can't do this checking in the test code.